README.md

```diff
@@ -22,9 +22,17 @@ In the specifications in the above table of contents, the keywords "MUST", "MUST
 
 To provide context for users the following section gives example use cases for each part of the spec.
 
-## Filesystem Bundle & Configuration
+**Container Configuration**
```

Comment:
I'd rather stick to headers (they go [all the way down to h6][1] ;), so we get link anchors.

[1]: http://daringfireball.net/projects/markdown/syntax#header